docs/reference/ilm/ilm-with-existing-indices.asciidoc

```diff
@@ -3,13 +3,16 @@
 [[ilm-with-existing-indices]]
 == Manage existing indices
 
+NOTE:  When migrating from Curator, ensure you are running Curator version 5.7 or greater
+so that ILM indices will be ignored as you phase out existing indices.  Otherwise, 
+you may choose to manage existing indices with ILM in the manner described in this 
+section.
```

Comment:
```suggestion
NOTE:  If migrating from Curator, ensure you are running Curator version 5.7 or greater
so that Curator will ignore ILM managed indices.
```

Reply:
@PhaedrusTheGreek - did you mean to accept the suggestion ?  If so click the Commit suggestion button (not resolve conversation button).